CHANGES.md

```diff
@@ -46,6 +46,10 @@ creating a new release entry be sure to copy & paste the span tag with the
 `actions:bind` attribute, which is used by a regex to find the text to be
 updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
+-------------------------------------------------------------------------------
+## __cylc-8.0b2 (<span actions:bind='release-date'>Released 2021-??-??</span>)__
+
+
 -------------------------------------------------------------------------------
 ## __cylc-8.0b1 (<span actions:bind='release-date'>Released 2021-04-21</span>)__
```

Comment:
I removed this in Cylc UI I think, because I think the GH Action looks for this string to replace by the date (although not sure if having it more than once actually breaks it or not). https://github.com/cylc/release-actions/blob/c4814ec1966191dcd359654a0684e1affeb3777d/stage-1/update-changelog-release-date/action.yml#L22

Reply:
I took this at its word :grin: :+1: 
https://github.com/cylc/cylc-flow/pull/4192/files#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2L44-R48

Reply:
That link doesn't work...

Reply:
```markdown
  44 <!-- The topmost release date is automatically updated by GitHub Actions. When
  45 creating a new release entry be sure to copy & paste the span tag with the
  46 `actions:bind` attribute, which is used by a regex to find the text to be
  47 updated. Only the first match gets replaced, so it's fine to leave the old
  48 ones in. -->
```

Reply:
#TodayILearned :grimacing:

Reply:
He heh, no it was a good question!